ruby/bps-stan/lib/bps/stan.rb

```diff
@@ -0,0 +1,11 @@
+require 'bps'
+require 'bps/publisher/stan'
+
+module BPS
+  module Publisher
+    register('stan') do |url, **opts|
+      cluster_id, client_id, url_opts = STAN.parse_url(url)
+      STAN.new(cluster_id, client_id, **STAN.coercer.coerce(url_opts.merge(opts)))
```

Comment:
it would be nice to url_opts.update(opts):

1. already too much going on in the next line
2. we can safely update instead of allocating with merge